### DISCUSSION TITLE
以太幣市值即將超越比特幣?PoW?PoS?

### DISCUSSION THREAD

**Discussion by williamyen0623** [1101-GS4719-C/Python-Community-GS4719-C#706] [科技新知]
大部分的人看以太幣單顆價值遠小於比特幣, 但事實上以太幣的總市值即將超越比特幣, 這篇文章講述了為何比特幣漸漸成為不可持續的投資以及像BTC這種Proof of Work的運作機制所遇到的瓶頸(ETH將轉型成Proof of Stake)
https://www.blocktempo.com/why-eth-flippening-btc-is-good-for-crypto/

**Comment by Yunhans** [comment]
要all in以太幣了嗎

**Reply by williamyen0623** [reply]
在還沒轉型時大家預測ETH轉型PoS的時間而去投資的話, 會對ETH的價格造成不好的影響, 但現在已經合併成功
https://technews.tw/2022/09/15/ethereum-2-merge-done/
或許是個不錯的投資項目, 以上不是任何投資建議, 虛擬貨幣風險極高, 請理性投資

**Comment by Chiway1229** [comment]
比特幣我覺得歸零的可能性偏低，但下降的趨勢正在逐漸實現
好耶 開始做空囉

**Comment by danwan1027** [comment]
我 all in

**Comment by evanchou1204** [comment]
偏專業，樓主準備接下一任區塊鏈研究社社長了嗎

**Comment by Pietra1226** [comment]
超猛，但我還是不敢踏進這塊區域QQ

**Comment by manipulateed** [comment]
以太幣的獎勵機制增加了選擇此虛擬貨幣的流量，會有將超越比特幣的總市值也指日可待。

**Comment by harrylee1971** [comment]
快all in，下半輩子不是大富大貴就是火車站見
